src/lib.rs

```diff
@@ -451,6 +451,26 @@ impl<K: Hash + Eq, V, S: BuildHasher, A: Clone + Allocator> LruCache<K, V, S, A>
         }
     }
 
+    pub fn get_many<'a, 'b, Q>(&mut self, ks: impl Iterator<Item = &'b Q>) -> Vec<Option<&'a V>>
```

Comment:
```suggestion
    pub fn get_many<'a, 'b, Q>(&'a mut self, ks: impl Iterator<Item = &'b Q>) -> Vec<Option<&'a V>>
```

Otherwise the returned lifetime can be arbitrary long and there could be potentially use-after-free if we `insert` after that, IIUC.